attractions/table/table.svelte

```diff
@@ -0,0 +1,74 @@
+<script>
+  import Label from '../typography/label.svelte';
+
+  /**
+   * @typedef {{
+   *  text: string;
+   *  value: string;
+   *  align?: "start" | "center" | "end";
```

Comment:
I just realized, I don't really like `start` and `end`:
* `left` and `right` would fit better with `center`
* what's a start for me is the end for you, Mr. RTL

Reply:
> what's a start for me is the end for you, Mr. RTL

that is precisely the reason they are named as such. `start` and `end` were added to [`text-align`](https://developer.mozilla.org/en-US/docs/Web/CSS/text-align) in CSS3 to adapt to the language direction. `right` and `left` are rigid, while `start` and `end` respect the [`direction`](https://developer.mozilla.org/en-US/docs/Web/CSS/direction) setting